src/rpc/util.h

```diff
@@ -154,21 +154,24 @@ struct RPCArg {
         /** Required arg */
         NO,
         /**
+         * The arg is optional for one of two reasons:
+         *
          * Optional arg that is a named argument and has a default value of
-         * `null`. When possible, the default value should be specified.
-         */
-        OMITTED_NAMED_ARG,
-        /**
+         * `null`.
+         *
          * Optional argument with default value omitted because they are
-         * implicitly clear. That is, elements in an array or object may not
```

Comment:
Is there a reason that "or object" was removed from here? My understanding is that `RPCArg::Optional::OMITTED` still applies to elements in objects, such as here
https://github.com/bitcoin/bitcoin/blob/392dc68e37be9fc7adb32496b13d9b50262e317d/src/wallet/rpc/backup.cpp#L1258-L1293

Reply:
"or object" is covered by the first of the two reasons:

> Optional arg that is a named argument and has a default value of null


(named arguments are basically keys in a dictionary/object)

Reply:
Thank you, the root of my confusion was the definition of named arguments.

Reply:
If there are improvements, they can be appended to #26919, which is a follow-up